changelogs/fragments/bump_to_215.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Can we also add a `release_summary` section like this: https://github.com/ansible-collections/cisco.ios/pull/1082/files#diff-a79e8a20ba9c6a3ef4751f43f85fa9c8f4bb09d29c1a122491c2fdb46a3d1468R1-R4?